2021/6/23のコミットからテストが失敗している

## 起こっている問題 / The Problem
- build の install dependencies で失敗している様子
https://github.com/code-for-hamamatsu/covid19/runs/2893311897

```
 [1/5] Validating package.json...
error covid19@1.0.0: The engine "node" is incompatible with this module. Expected version "14.17.0". Got "14.17.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

Nodeのバージョンが古くなっているため？？

## スクリーンショット / Screenshot

![Commits_·_code-for-hamamatsu_covid19](https://user-images.githubusercontent.com/662084/127411758-782d036e-d543-4089-a5cf-7fa270dbc834.png)


## 期待する見せ方・挙動 / Expected Behavior
- テストが失敗しないこと
- テストが失敗したことを通知する（このissueで対応する必要はない）

--- a/.github/workflows/e2etest.yml
+++ b/.github/workflows/e2etest.yml
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.17.0]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
 
     steps:

--- a/README.md
+++ b/README.md
@@ -53,6 +53,8 @@ $ yarn install
 $ docker-compose run --rm app yarn install
 ```
 
+起動後、[http://localhost:3000](http://localhost:3000) へアクセスする。
+
 ### VSCode + Remote Containersで開発する場合
 
 1. VSCodeの拡張機能「[Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)」を導入します。

--- a/package.json
+++ b/package.json
@@ -1,8 +1,8 @@
 {
   "name": "covid19",
   "version": "1.0.0",
-  "description": "東京都 新型コロナウイルス感染症対策サイト",
-  "author": "東京都",
+  "description": "浜松市 新型コロナウイルス感染症対策サイト",
+  "author": "浜松市",
   "private": true,
   "engines": {
     "node": "14.17.0"

